tests/Fixtures/filters/base64_encode.test

```diff
@@ -0,0 +1,10 @@
+--TEST--
+"base64_encode" filter
+--TEMPLATE--
+{{ "foo"|base64_encode|raw }}
```

Comment:
why using the `rawè filter here ?

Reply:
used the test from json_encode as reference